numpy/core/src/umath/ufunc_object.c

```diff
@@ -2199,10 +2362,10 @@ PyUFunc_GeneralizedFunction(PyUFuncObject *ufunc,
 
     /* Use remapped axes for generalized ufunc */
     int broadcast_ndim, iter_ndim;
-    int core_num_dims_array[NPY_MAXARGS];
-    int *core_num_dims;
+    int core_num_dims[NPY_MAXARGS];
```

Comment:
As per discussion in PR #11176, this should be renamed

Reply:
Need to think a bit about the logic of the names, since we're copying and adjusting two arrays: `core_num_dims` and `core_dim_sizes`. In both cases, the copies represent the *actual* number of dimensions that the operands have and the sizes that they imply. So, long versions could be
`actual_core_num_dims` and `actual_core_dim_sizes` - or remove `_core`.

Alternatively, I have been wondering whether it would make sense to have a mini-struct `actual` that had whatever parts of the ufunc would need changing (could be expanded to include `core_dim_ixs` for "calculated" output indices, e.g.). So that would mean `actual->core_num_dims`, `actual->core_dim_sizes`. But this could be done in a separate PR as well.

Reply:
p.s. I'm also not so happy that as is, `core_num_dims` gets copied even if in standard usage it never gets adjusted. But I guess it is only a few numbers, so very little overhead...